www/docs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@vltpkg/docs",
-  "version": "0.0.0",
+  "version": "0.0.0-1",
```

Comment:
This is to trigger a docs PR deploy to make sure `pnpm@10` works in the Vercel build.